src/app/pages/applications/catalog/catalog.component.html

```diff
@@ -16,8 +16,16 @@
           <div class="logo">
             <img [src]="item.icon_url" [src-fallback]="imagePlaceholder"/>
           </div>
-          <div class="badge-area">
-            <div class="partner-badge capitalized-text">
+          <div class="badge-area" style="background-color: var(--primary)" *ngIf="item.catalog.label == 'OFFICIAL'">
```

Comment:
We should put the styles in CSS classes

Reply:
I've moved it to an additional class that only overrides the color.